test/functionalTest/README.md

```diff
@@ -26,6 +26,8 @@ Next install accumulator-server.py depencencies:
 ```
 pip install Flask==1.0.2
 pip install pyOpenSSL==19.0.0
+pip install python-etcd==0.4.5
+pip install paho-mqtt==1.5.1
```

Comment:
```suggestion
pip install Flask==1.0.2
pip install pyOpenSSL==19.0.0
pip install python-etcd==0.4.5
pip install paho-mqtt==1.5.1
```
```suggestion
pip install Flask==1.0.2
pip install pyOpenSSL==19.0.0
pip install paho-mqtt==1.5.1
```

according to https://github.com/telefonicaid/fiware-orion/blob/master/ci/rpm8/build-dep.sh#L82-L84

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3955/commits/e4c64678323eba11780df2cf67e1633c3a160c67